components/teams-service/server/v2/server.go

```diff
@@ -22,12 +24,24 @@ func NewServer(service *service.Service) *Server {
 	return &Server{service: service}
 }
 
+// GetVersion returns the version of Teams GRPC API
```

Comment:
I wonder if we make use of this outside of the smoke tests... 🤔

Reply:
we don't atm 🤷‍♀️

Reply:
I think we should be using the health protocol for health checks 😅 

This endpoint is pretty useless, given that we never shipped updates to services in isolation (only complete A2 versions). **But** 🤷‍♂️ the endpoint doesn't hurt.

Reply:
https://github.com/chef/automate/blob/31442fe17fe52e537809ca6bf9ea21eb729f636a/components/teams-service/server/server.go#L34 this one